packages/polaris-viz/src/hooks/useWatchActiveSeries.ts

```diff
@@ -7,18 +7,34 @@ import {getEventName} from './ColorVisionA11y';
 import {useCallbackRef} from './useCallbackRef';
 
 export function useWatchActiveSeries(
-  id: string,
+  id: string | null,
```

Comment:
The point of adding the separate `setActiveSeriesListener()` is so we don't need to make this optional right? So let's remove that logic?